service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GpcWebClientTest.java

```diff
@@ -230,6 +232,27 @@ public void When_GetStructuredRecord_With_HttpStatus5xx_Expect_RetryWithGpcServe
         verify(gpcTokenBuilder, times(FOUR)).buildToken(taskDefinition.getFromOdsCode());
     }
 
+    @Test
+    void When_GetStructuredRecord_With_HttpStatus200_Expect_AuthorizationHeaderToBePresent() throws InterruptedException {
+        // given
+        final GetGpcStructuredTaskDefinition taskDefinition = getStructuredDefinition();
+        final Collection<String> tokens = new ArrayList<>();
+        final Pattern jwtPattern = Pattern.compile("^[a-zA-Z0-9+/]*={0,2}\\.[a-zA-Z0-9+/]*={0,2}\\.[a-zA-Z0-9+/]*={0,2}$");
+
+        // when
+        for(int i = 0; i < FOUR; i++) {
+            mockWebServer.enqueue(STUB_OK);
+            gpcWebClient.getStructuredRecord(taskDefinition);
+            tokens.add(mockWebServer.takeRequest().getHeader(HttpHeaders.AUTHORIZATION));
+        }
+
+        // then
+        assertThat(tokens).hasSize(FOUR);
+        tokens.stream()
+                .map(token -> token.split("Bearer")[1].trim())
+                .forEach(token -> assertThat(token).matches(jwtPattern));
```

Comment:
I'd probably assert that the token had come from the the `gpcTokenBuilder.buildToken` method, as then it's clearer how its being generated, plus this test becomes less coupled to the specifics of JWT.